types/svelte-range-slider-pips/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode": "code",
+                "errors": [["NeedsExportEquals", false]]
```

Comment:
Looking at the source JS, I think this actually does need `export =` syntax not `export default`. If you `npm i svelte-range-slider-pips` and run node with `require('svelte-range-slider-pips')`, you get a class with no `default` property.